docs/ServiceNow-connector.md

```diff
@@ -0,0 +1,27 @@
+# ServiceNow Connection
+
+Description
+-----------
+Use this connection to access data in ServiceNow.
+
+Properties
+----------
+**Name:** Name of the connection. Connection names must be unique in a namespace.
+
+**Description:** Description of the connection.
+
+**Client ID**: The Client ID for ServiceNow Instance.
+
+**Client Secret**: The Client Secret for ServiceNow Instance.
+
+**REST API Endpoint**: The REST API Endpoint for ServiceNow Instance. For example, `https://instance.service-now.com`
```

Comment:
Can you remove the link in the URL since it's not real?

Reply:
Removed